x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/create_rule/route.ts

```diff
@@ -56,11 +56,10 @@ export const createRuleRoute = (router: SecuritySolutionPluginRouter): void => {
             'lists',
             'actions',
           ]);
-
+          const actionsClient = ctx.actions.getActionsClient();
           const rulesClient = ctx.alerting.getRulesClient();
-          const detectionRulesClient = ctx.securitySolution.getDetectionRulesClient();
```

Comment:
Can we pass this dependency inside of `x-pack/plugins/security_solution/server/request_context_factory.ts` to avoid doing it in every route handler?

Reply:
@banderror I think you might be looking at an outdated version of the file. We do that here: https://github.com/elastic/kibana/pull/183937/files#diff-74701f6237f2551aeb4642920e86b13c2bd8a4d046e74885b1c818606c7bd26b

Thanks to @jpdjere and @nikitaindik for suggesting that change.

Reply:
Ah, ok! Weird. Thanks for fixing that.